rele/client.py

```diff
@@ -15,53 +15,51 @@
 
 class Subscriber:
 
-    def __init__(self):
+    def __init__(self, gc_project_id, credentials):
```

Comment:
Subscriber and Publisher has a pubsub client as colaborator.

Instead of passing credentials through the constructor, it would be cleaner to extract client's instantiation to an static function like `get_pubsub_client()` and either inject the client (oop) or instantiate in the constructor by doing `self._client = get_pubsub_client()`(pythonic).